docs/design/input.md

```diff
@@ -4,6 +4,8 @@ Github audit plugin will be using config.yaml file for taking input from user
 
```

Comment:
Comments should be above the field 
## API token in base64 encode format. <REQUIRED/OPTIONAL>, default value: 
access_token: <>

Reply:
done